lib/unzip.js

```diff
@@ -23,7 +23,9 @@ module.exports = function (input, isGZip, isSprite, callback) {
     }
     return JSZip.loadAsync(input)
         .then(function (zip) {
-            return zip.file(isSprite ? 'sprite.json' : 'project.json').async('string')
+            const dir = Object.values(zip.files).filter(f => f.dir);
+            const projectJson = dir.length === 1 ? dir[0].name + 'project.json' : 'project.json';
```

Comment:
Hmmmm... I think I would prefer for the parser to look for the project.json in a dir only if it fails to find a file with the name at the top level. Also, I think we should probably make the same provisions for sprite.json as we do for project.json.

I am not sure if it is possible to find out if zip.file('project.json') actually succeeded or not...